index.js

```diff
@@ -1,20 +1,28 @@
 'use strict';
 
 var defined = require('defined');
+var through = require('@ljharb/through');
+
 var createDefaultStream = require('./lib/default_stream');
 var Test = require('./lib/test');
 var Results = require('./lib/results');
-var through = require('@ljharb/through');
 
 var canEmitExit = typeof process !== 'undefined' && process
+	// @ts-expect-error i think old browserify uses `process.browser`
```

Comment:
`// @ts-expect-error` is a very blunt tool. It's almost always better to use something more narrow, like an `as any`, if possible. `// @ts-expect-error` suppresses much more and so is more dangerous.

Here and in the other uses.

Reply:
hmm - i'd definitely prefer to be able to tell TS i only expect a specific kind of error, but i'm pretty allergic to any use of `any`, and casting seems much more dangerous to me

Reply:
Yeah that would be a nice feature. Blocked on https://github.com/microsoft/TypeScript/issues/45937 - the errors aren't categorized.

A type assertion with `any` is strictly safer than `ts-expect-error`. +1 to being allergic to both/either, but in this case, you're losing type safety by using the directive comment.